runner/src/main/kotlin/pl/edu/prz/kod/runner/domain/executor/PythonExecutor.kt

```diff
@@ -6,7 +6,7 @@ import pl.edu.prz.kod.runner.domain.Language
 class PythonExecutor: AbstractExecutor(Language.PYTHON) {
 
     override fun execute(code: String): ExecutionResult {
-        val escapedCode = code.replace("\"", "\\\"")
+        val escapedCode = code.replace("\\", "\\\\").replace("\"", "\\\"")
```

Comment:
This looks like a good candidate to export to a function in AbstractExecutor imo